dangerzone/gui/main_window.py

```diff
@@ -448,49 +487,83 @@ def __init__(self, dangerzone: DangerzoneGui) -> None:
             log.error(str(e))
             state = "not_installed"
 
         else:
-            # Can we run `docker image ls` without an error
+            # Can we run `docker/podman image ls` without an error
             with subprocess.Popen(
                 [container_runtime, "image", "ls"],
                 stdout=subprocess.DEVNULL,
-                stderr=subprocess.DEVNULL,
+                stderr=subprocess.PIPE,
                 startupinfo=get_subprocess_startupinfo(),
             ) as p:
-                p.communicate()
+                _, stderr = p.communicate()
                 if p.returncode != 0:
-                    log.error("Docker is not running")
+                    log.error(f"{runtime_name} is not running")
                     state = "not_running"
+                    error = stderr.decode()
                 else:
                     # Always try installing the container
                     state = "install_container"
 
         # Update the state
-        self.state_change(state)
+        self.state_change(state, error)
 
-    def state_change(self, state: str) -> None:
+    def state_change(self, state: str, error: Optional[str] = None) -> None:
         if state == "not_installed":
-            self.label.setText(
-                "<strong>Dangerzone Requires Docker Desktop</strong><br><br><a href='https://www.docker.com/products/docker-desktop'>Download Docker Desktop</a>, install it, and open it."
-            )
+            if platform.system() == "Linux":
+                self.label.setText(
+                    (
+                        "<strong>Dangerzone requires Podman</strong><br><br>"
+                        "Install it and retry."
+                    )
+                )
+            else:
+                self.label.setText(
+                    (
+                        "<strong>Dangerzone requires Docker Desktop</strong><br><br>"
+                        "<a href='https://www.docker.com/products/docker-desktop'>Download Docker Desktop</a>"
+                        ", install it, and open it."
+                    )
+                )
             self.buttons.show()
         elif state == "not_running":
-            self.label.setText(
-                "<strong>Dangerzone Requires Docker Desktop</strong><br><br>Docker is installed but isn't running.<br><br>Open Docker and make sure it's running in the background."
-            )
+            if platform.system() == "Linux":
+                # "not_running" here means that the `podman image ls` command failed.
+                message = (
+                    "<strong>Dangerzone requires Podman</strong><br><br>"
+                    "Podman is installed but cannot run properly. See errors below"
+                )
+                if error:
+                    self.traceback.set_content(error)
+
+                self.label.setText(message)
+
+            else:
+                self.label.setText(
+                    (
```

Comment:
The extra parentheses are not needed, I believe.

Reply:
Thanks, removed.